GCTC-NTGC/gc-digital-talent
# [Dep] Bump storybook and addons to 7.3.1

## 👋 Introduction

Bump all storybook deps to latest version.

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Run storybook `npm run storybook`
2. Confirm everything works
